core/tests/protractor/loginFlow.js

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests to login, check various pages
+ * and then logout.
+ */
+
+var LearnerDashboardPage = require(
+  '../protractor_utils/LearnerDashboardPage.js');
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var waitFor = require('../protractor_utils/waitFor.js');
+
+describe('Login flow', function() {
```

Comment:
Looks more like a test for the profile dropdown menu to me...

Reply:
We have another nested `describe` block for profile dropdown menu [here](https://github.com/oppia/oppia/pull/5340/files/6a539747acef7724add10bd9543fb3f8eeda6a8b#diff-aa9e49320b334ac001383c1b87962f0fR42).  
Do we still need to change this description?

Reply:
Could we rename it (and the file) to something like "navigation" perhaps? This file is, like, 90% non-login stuff, so the name seems misleading.

Reply:
Yes, I agree. Since we already have a file named navigation for mobile, shall I change this to profileFlow since it's mostly concerned about navigation using the profile menu?

Reply:
profileMenuFlow perhaps? But otherwise sgtm.

Reply:
Done, renamed to profileMenuFlow. Thanks!